### DISCUSSION TITLE
Orange Pi Zero & Airplay 2 > Success !

### DISCUSSION THREAD

**Discussion by beber666** [mikebrady/shairport-sync#1645] [General]
Just want to thanks the team for your great work

I've been trying since multiple months to make a simple multiroom system (volumio balena etc...)
I've got intergated speakers in my rooms, and my idea were to create a home made multi room system

As I don't want to spend too much money in raspberries, I've tried with Orange Pi Zero LTS (with H3 chip and 512 Mb RAM)
It's a very low cost card, and I'm using it with a 2$ USB DAC

I'm using Armbian_23.02.2_Orangepipcplus_jammy_current_5.15.93

And all works like a charm, Airplay 2 included !
So thanks again the team to make this possible !!

**Comment by Francisk0** [comment]
Hi, 
Can you give more details?

**Comment by beber666** [comment]
You need to install first Armbian_23.02.2_Orangepipcplus_jammy_current_5.15.93 and follow the guide to deploy shairport-sync

You need also to do the trick for the alsamixer https://f1atb.fr/fr/mise-en-route-orange-pi-zero/ (it's french but you can easily translate)
